src/main/java/com/icthh/xm/tmf/ms/activation/service/RetryService.java

```diff
@@ -54,8 +58,13 @@ public void retry(SagaEvent sagaEvent, SagaTaskSpec sagaTaskSpec, SagaEvent.Saga
         sagaEvent.setRetryNumber(sagaEvent.getRetryNumber() + 1);
 
         if (sagaEvent.getRetryNumber() > sagaTaskSpec.getRetryCount() && sagaTaskSpec.getRetryCount() >= 0) {
-            log.warn("Limit retry exceeded for event {}. {} > {}", sagaEvent, sagaEvent.getRetryNumber(),
-                     sagaTaskSpec.getRetryCount());
+            log.warn("Retry limit exceeded for event {}. {} > {}", sagaEvent, sagaEvent.getRetryNumber(),
+                sagaTaskSpec.getRetryCount());
+            try {
+                self.retryLimitExceeded(sagaEvent, sagaTaskSpec, eventStatus);
+            } catch (Throwable e) { // Because of fact that groovy code can have compilation errors
+                log.error(" Error unable to start compensation lep ", e);
```

Comment:
suggest changing log to :
`log.error("Error unable to start compensation lep: {}", e.getMessage(), e);`